src/cmdline/mod.rs

```diff
@@ -337,6 +337,18 @@ impl From<Cmdline> for Vec<u8> {
     }
 }
 
+impl fmt::Debug for Cmdline {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{}", self.line)
+    }
+}
+
+impl PartialEq for Cmdline {
+    fn eq(&self, other: &Self) -> bool {
+        self.line == other.line
```

Comment:
Why not check the capacity as well? In which case we could also just derive partialeq.

Reply:
It looked more natural to not involve the capacity in checks for equality (i.e. the same as happens for `Vec` IIRC). I can also add another derive if there's a strong preference for that.